packages/beacon-node/src/sync/interface.ts

```diff
@@ -86,7 +86,7 @@ export type UnknownBlock = PendingBlockCommon & {
 export type UnknownBlockInput = PendingBlockCommon & {
   status: PendingBlockStatus.pending | PendingBlockStatus.fetching;
   parentBlockRootHex: null;
-  blockInput: BlockInput & {type: BlockInputType.blobsPromise};
+  blockInput: (BlockInput & {type: BlockInputType.blobsPromise}) | NullBlockInput;
```

Comment:
need to find a way to distinguish different types (missing blobs vs block) for this metric `lodestar_sync_unknown_block_requests_total` https://github.com/ChainSafe/lodestar/blob/1d3bf0412ab8ac169f8cbcdfabcbc8505e3e5cc7/packages/beacon-node/src/sync/unknownBlock.ts#L113